blog/june-2019-workshop.html

```diff
@@ -0,0 +1,275 @@
+  <a href="https://dawn.cs.stanford.edu/2019/06/15/superglue/">blog post on slicing in SuperGLUE</a>
+  and the slicing
+  <a href="#workshop-contents">workshop materials</a> below.
+
+  <h3>Upgraded labeling pipeline</h3>
+
+  The programmatic labeling engine is getting updated to the latest and greatest.
+  The new core generative label model uses the matrix-completion approach studied in our
+  <a href="https://arxiv.org/abs/1810.02840">AAAI 2019</a> and
+  <a href="https://arxiv.org/abs/1903.05844">ICML 2019</a> 
+  papers and implemented in
+  <a href="https://github.com/HazyResearch/metal/">Snorkel MeTaL</a>.
+  The label model is currently implemented in PyTorch, but we’re starting work on a
+  TensorFlow-based and TFX servable version that many workshop participants requested.
+
+  <br><br>
+
+  We're also providing native support for applying labeling functions using Spark and Dask,
+  making it easier to work with massive data sets.
+  Spark integration in particular has been a common feature request, and this was echoed
+  by the workshop attendees.
+
+  <h3>General purpose engine for data set creation</h3>
+
+  We redesigned Snorkel in version 0.9 to be a general purpose engine
+  for programmatically building and managing training data sets across different domains
+  and ML problem types.
+  TODO: EXTRA SENTENCE HERE?
+  We’re keeping the core programmatic labeling functionality from the current information
+  extraction application version, but removing the extraction-specific tooling and database
+  from the central library.
+  Workshop attendees were excited about this change, as they could see how to apply Snorkel to their
+  ML use cases - whether they used text, image, or tabular data - in a straightforward way
+  and integrate with their existing data infrastructure.
+
+  <br><br>
+
+  Many of our workshop participants also expressed their interest in continued support of the
+  current information extraction application version of Snorkel.
+  We’ll absolutely be doing this, hosted in a new GitHub repo and upgraded with the new
+  label model from the central Snorkel library.
+  Our information extraction system is just one example of a Snorkel-powered
+  end-to-end application.
+  We'll be releasing a host of other tutorials and examples in the next couple months.
+  They’ll have a new home in a dedicated tutorials repo, where you'll be able to see Snorkel
+  applied to a wide variety of ML problems.
+
+  <h3>A modern Python library</h3>
+
+  Workshop participants were excited to see modern installation support with pip and conda, as well
+  as lots and lots of unit and integration testing.
+  We also completed full API documentation - requested by everyone at the workshop - during a sprint
+  this past week.
+  All of this will make it easier to get started with, receive support, and contribute to Snorkel.
+
+  <br><br>
+
+  Stay tuned for the full Snorkel v0.9 release post, where we’ll share more details and resources!
```

Comment:
Can join the Google Group (https://groups.google.com/forum/#!forum/snorkel-ml) to stay tuned.

Reply:
Linked at the bottom, probably enough